### PR TITLE
Portfolio: Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by ger619**
For the third milestone of your portfolio website, you will:

- Add a contact form to your portfolio and the portfolio of your coding partner.
- Make sure that you guide your users' experience in a way that helps them fill in the form without frustration.
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -205,3 +205,87 @@ img {
...
+  font-size: 17px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.form-style button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
```
Comment (sja-thedude) [inline main.css]: - [ ] Kindly use the flex box in the contact section to layout elements as per requiremnets.

File: main.css [hunk 2]
```diff
@@ -205,3 +205,87 @@ img {
   color: #344563;
   margin-top: 60px;
 }
+
+.footer {
+  background-color: #6070ff;
+  min-height: 779px;
+  max-width: 375px;
+  border-radius: 80px 0 0 0;
+  background-image: url(./images/background.png);
```
Comment (sja-thedude) [inline main.css]: - [ ] Kindly make sure to follow the design template as given in Figma. As you have noticed your background image is not properly placed as given. Fix it.
- Your design:
Screenshot 2022-03-22 at 10 55 20 PM

- Figma design:
Screenshot 2022-03-22 at 11 08 59 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ger619, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Linters are passing
✔️ Descriptive PR title and description

## Required 
- [ ] Kindly make sure to name your branch according to your feature name
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css (hunk 2 above)
Comment (ger619) [reply main.css]: Done

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ger619,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**